app/configurations/config.hb.js

```diff
@@ -315,8 +315,39 @@ export default configMerger(walttiConfig, {
           en: 'Charging stations',
           de: 'Ladestationen',
         },
-        url: '/assets/geojson/hb-layers/charging.geojson',
+        url: GEOJSON_LAYERS_BASE_URL + 'charging.geojson',
+      },
+      // Bike rental places in Stuttgart
+      {
+        name: {
+          fi: '',
+          en: 'Bicycle rental places',
+          de: 'Fahrradverleih',
+        },
+        url: GEOJSON_LAYERS_BASE_URL + 'bikerental.geojson',
       },
+      // park and ride places
+      {
+        name: {
+          fi: '',
+          en: 'Park and Ride',
+          de: 'Park-Und-Ride',
+        },
+        url: GEOJSON_LAYERS_BASE_URL + 'parkandride.geojson',
+      }
+      /*,
```

Comment:
Is this (ignored anyway) layer necessary? I think it should be part of bicycle infrastructure and hence not written here. Additionally, the url would be wrong.